sdk/storage/storage-blob/src/BlobServiceClient.ts

```diff
@@ -336,6 +336,16 @@ export class BlobServiceClient extends StorageClient {
    *                                                  AnonymousCredential is used.
    * @param {StoragePipelineOptions} [options] Optional. Options to configure the HTTP pipeline.
    * @memberof BlobServiceClient
+   *
+   * @example
+   * ```js
+   * const defaultAzureCredential = new DefaultAzureCredential();
```

Comment:
Can we add an inline comment about how to find `DefaultAzureCredential`? Because it's not from storage SDK.

Reply:
Yup, please add, @willmtemple  :)

Reply:
Yes, good catch. I will do the same in storage-queue as well on the other PR.